lib/manager/flux/extract.ts

```diff
@@ -0,0 +1,123 @@
+import { loadAll } from 'js-yaml';
+import { HelmDatasource } from '../../datasource/helm';
+import { logger } from '../../logger';
+import { SkipReason } from '../../types';
+import { readLocalFile } from '../../util/fs';
+import type { ExtractConfig, PackageDependency, PackageFile } from '../types';
+import type {
+  FluxManifest,
+  FluxResource,
+  HelmRelease,
+  HelmRepository,
+} from './types';
+
+function readManifest(content: string): FluxManifest {
```

Comment:
```suggestion
function readManifest(content: string): FluxManifest | null {
```

Please add `lib/flux` to `tsconfig.strict.json`
If there are transitive errors, only try to fix issues in flux and remove from strict afterwards. 🤗

Reply:
I added `lib/manager/flux/**/*.ts` to `tsconfig.strict.json` and ran `yarn build` and it succeeded - I'm guessing that means we're good...if we're not I'm sure the build will yell at me shortly. 🙂

Reply:
Oh, I had to run `yarn null-check`...the output from the broken build only shows a few warnings in `flux/extract.spec.ts`, so I think we're good here.

Reply:
If no errors occured, please add and commit again.